draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -422,9 +453,47 @@ disclosable array elements for which they did not receive a Disclosure. In the
 example above, the verification process would output an array with only one
 element unless a matching Disclosure for the second element is received.
 
-## Example 1: SD-JWT {#example-1}
 
-In this example, a simple SD-JWT is demonstrated.
+## Key Binding JWT {#kb-jwt}
+
+This section defines the contents of the optional Key Binding JWT, which
+the Holder MAY include in the SD-JWT to prove the Key Binding to the Verifier.
+
+The JWT MUST contain the following elements:
+
+* in the JOSE header,
+    * `typ`: REQUIRED. MUST be `kb+jwt`, which explicitly types the Key Binding JWT as recommended in Section 3.11 of [@!RFC8725].
+    * `alg`: REQUIRED. A digital signature algorithm identifier such as per IANA "JSON Web Signature and Encryption Algorithms" registry. MUST NOT be `none` or an identifier for a symmetric algorithm (MAC).
+* in the JWT payload,
+    * `iat`: REQUIRED. The value of this claim MUST be the time at which the Key Binding JWT was issued using the syntax defined in [@!RFC7519].
+    * `aud`: REQUIRED. The intended receiver of the Key Binding JWT. How the value is represented is up to the protocol used and out of scope of this specification.
+    * `nonce`: REQUIRED. Ensures the freshness of the signature. The value type of this claim MUST be a string. How this value is obtained is up to the protocol used and out of scope of this specification.
+
+To validate the signature on the Key Binding JWT, the Verifier MUST use the key material in the SD-JWT. If it is not clear from the SD-JWT, the Key Binding JWT MUST specify which key material the Verifier needs to use to validate the Key Binding JWT signature using JOSE header parameters such as `kid` and `x5c`.
+
+Below is a non-normative example of a Key Binding JWT header:
+
+```
+{
+  "alg": "ES256",
+  "typ": "kb+jwt"
+}
+```
+
```

Comment:
With the other examples gone from this section - this one feels a bit out of place. And not needed. 
```suggestion
```

Reply:
should this example be re-added to another section? it was added because people wanted to see an example of a KB-JWT header..

Reply:
I agree that removing makes most sense. We're not showing any other headers in the document, there is nothing surprising in this header, and people can still decode the KB-JWT if they like to see a header.